backend/tests/apps/github/api/search/user_test.py

```
@@ -0,0 +1,51 @@
+"""Test cases for the get_users function in the GitHub user search API module."""
```

Ruff (0.11.9)

File `backend/tests/apps/github/api/search/user_test.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)